constants.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  DEFAULT_PROJECT_NAME_KEY: "untitledProject"
+  DEFAULT_PROJECT_NAME_KEY: "Project zonder naam"
```

Comment:
This is the key, not the actual words to use. It's used to get the translation, you can find it there https://github.com/StudioKrok/thimble.mozilla.org/blob/f1347eb2fffd02aa23eb8916bfe68fcfaa9499bb/locales/nl/server.properties#L408.

Reply:
You might want to change this tho https://github.com/StudioKrok/thimble.mozilla.org/blob/master/default/empty-project/index.html